Makefile.inc

```diff
@@ -94,3 +94,16 @@ endif
 
 CODEOWNERS: CODEOWNERS.in
 	$(SCRIPTS_DIR)/gen-codeowners
+
+# This removes all Submariner-provided images and all untagged images
+# Use this to ensure you use current images
+# Copied from Makefile to provide this everywhere (until we can share
```

Comment:
What's the problem with running it in dapper always?

Reply:
We can’t remove the Dapper image if we’re running in Dapper.

Reply:
Right kind of chicken and egg, though when tested locally on *submariner* it did remove the shipyard base image (but failed to remove the container image since it's running, maybe thats what you were seeing)?

Reply:
Yes, the Dapper image, not the Shipyard base image.

Reply:
Theoretically it can be identified as such
```bash
set +m
shopt -s lastpipe
docker ps | grep $HOSTNAME | read _ dapper_image _
```

And then you can skip trying to delete `$dapper_image`

Reply:
That’s not what I’m trying to fix. I don’t care about skipping the image we’re running in, `docker rmi` does that for us, and since we’re not running with `set -e`, it doesn’t break out of the task. I would like to support running all this without Dapper everywhere at some point so that we _can_ delete the Dapper image.